tests/framework/CodeInterfaceTests/Scale/test_scale_combined_forward.xml

```diff
@@ -11,6 +11,7 @@
     </description>
     <revisions>
       <revision author="alfoa" date="2018-04-04">Test created</revision>
+      <revision author="alfoa" date="2019-03-07">Modified syntax in Functions as for issue #934</revision>
```

Comment:
I wonder if we could add these revisions via the conversion script, especially if the message is always going to be the same? Seems like it shouldn't be too hard. No change needed, just for thought.

Reply:
yes... you are right... I thought about it but later I forgot and did by hand O_o